.github/workflows/vcpkg_docker_amd64.yml

```diff
@@ -24,9 +24,7 @@ jobs:
         # arch is to tag docker images for uniqueness
         host:
           - { name: 'ubuntu-20.04', arch: '' }
-          - { name: ['self-hosted', 'Linux', 'ARM64'], arch: '_arm64' }
         container:
-          - { distro: 'ubuntu', version: '18.04' }
```

Comment:
We'll have to remember to change other repos that depend on the 18.04 image to just use 20.04 now